test/spec/core/CanvasSpec.js

```diff
@@ -2337,6 +2337,65 @@ describe('Canvas', function() {
     });
 
 
+    describe('#removePlane', function() {
+
+      it('should expect a plane', inject(function(canvas) {
+
+        expect(function() {
+
+          // when
+          canvas.removePlane();
+        }).to.throw('must specify a plane');
+      }));
+
+
+      it('should remove the plane', inject(function(canvas) {
+
+        // given
+        var plane = canvas.createPlane('a');
+
+        // when
+        canvas.removePlane(plane);
+
+        // then
+        expect(canvas.getPlane('a')).not.to.exist;
+      }));
+
+
+      it('should not remove canvas data-element-id', inject(function(canvas) {
+
+        // given
+        var rootA = { id: 'A' };
+        var rootB = { id: 'B' };
+
+        canvas.createPlane('a', rootA);
+        canvas.createPlane('b', rootB);
+
+        canvas.setActivePlane('a');
+
+        // when
+        canvas.removePlane('b');
+
+        // then
+        expect(svgAttr(canvas._svg, 'data-element-id')).to.equal('A');
```

Comment:
Please always (also) test via high-level APIs: `ElementRegistry.get(...)`.

This is closest to how we'll use it (and our users will use it, too).